documentation/specs/Modal.md

```diff
@@ -271,3 +271,11 @@ function PageWithModal() {
 
 - `useDialog`, `useModalOverlay` from `react-aria`
 - `IntersectionObserver` for styling when header and footer are stuck, as denoted here: https://ryanmulligan.dev/blog/sticky-header-scroll-shadow/
+
+---
+
+## Versions
+
+| Version    | Owner           | Change                                              |
+| :--------- | :-------------- | :-------------------------------------------------- |
+| 2023-08-21 | stephenjwatkins | Add `<ModalContainer />` to support custom triggers |
```

Comment:
Saw your comment on this. I'm not sure if it's too much info to be added as documentation in Storybook but might be cool to have them living and attached

Reply:
i agree. i'll think about how we could do that